projects/mercury/src/metadata/views/MetadataViewTable.js

```diff
@@ -53,6 +53,17 @@ export const MetadataViewTable = (props: MetadataViewTableProperties) => {
     const visibleColumns = columns.filter(column => visibleColumnNames.includes(column.name));
     const dataLinkColumn = columns.find(c => c.type === 'dataLink');
     const isResourcesView = view === RESOURCES_VIEW;
+    const {checkboxes, setCheckboxState} = props;
+
+    // initialize checkboxes
+    if (idColumn && data && data.rows) {
+        data.rows.forEach(row => {
+            const key = row[idColumn.name][0].value;
+            if (checkboxes[key] === undefined) {
+                setCheckboxState(key, false);
+            }
+        });
+    }
```

Comment:
it would be better to wrap this check into useEffect hook with parameters, otherwise it keeps running this check and there are some rendering conflicts with other components (see browser console errors)